### ISSUE TITLE
NGL and NGLView are unable to render this pdb

### ISSUE THREAD

**Issue by hainm** [arose/ngl#126]
```
HETATM    1  C1  UNL     1       5.842   3.226  -0.572  1.00  0.00           C
HETATM    2  O1  UNL     1       5.678   1.925  -0.990  1.00  0.00           O
HETATM    3  C2  UNL     1       4.784   0.958  -0.614  1.00  0.00           C
HETATM    4  C3  UNL     1       4.903  -0.273  -1.256  1.00  0.00           C
HETATM    5  C4  UNL     1       4.035  -1.320  -0.948  1.00  0.00           C
HETATM    6  C5  UNL     1       3.052  -1.165   0.003  1.00  0.00           C
HETATM    7  C6  UNL     1       2.066  -2.237   0.238  1.00  0.00           C
HETATM    8  O2  UNL     1       1.748  -2.989  -0.882  1.00  0.00           O
HETATM    9  C7  UNL     1       0.905  -1.784   1.046  1.00  0.00           C
HETATM   10  C8  UNL     1       1.492  -1.009   2.238  1.00  0.00           C
HETATM   11  O3  UNL     1       1.969   0.194   1.667  1.00  0.00           O
HETATM   12  C9  UNL     1       2.928   0.024   0.653  1.00  0.00           C
HETATM   13  C10 UNL     1       3.791   1.073   0.320  1.00  0.00           C
HETATM   14  N1  UNL     1      -0.041  -0.912   0.397  1.00  0.00           N
HETATM   15  C11 UNL     1      -0.972  -0.519   1.434  1.00  0.00           C
HETATM   16  C12 UNL     1      -2.196   0.157   0.927  1.00  0.00           C
HETATM   17  C13 UNL     1      -2.904  -0.589  -0.172  1.00  0.00           C
HETATM   18  O4  UNL     1      -3.563  -1.720   0.324  1.00  0.00           O
HETATM   19  C14 UNL     1      -1.926  -0.880  -1.221  1.00  0.00           C
HETATM   20  C15 UNL     1      -0.647  -1.520  -0.734  1.00  0.00           C
HETATM   21  C16 UNL     1      -3.935   0.369  -0.684  1.00  0.00           C
HETATM   22  C17 UNL     1      -3.558   1.688  -0.965  1.00  0.00           C
HETATM   23  C18 UNL     1      -4.487   2.709  -1.015  1.00  0.00           C
HETATM   24  C19 UNL     1      -5.805   2.441  -0.795  1.00  0.00           C
HETATM   25  F1  UNL     1      -6.747   3.430  -0.672  1.00  0.00           F
HETATM   26  C20 UNL     1      -6.200   1.158  -0.553  1.00  0.00           C
HETATM   27  C21 UNL     1      -5.282   0.138  -0.499  1.00  0.00           C
HETATM   28  H1  UNL     1       6.943   3.331  -0.316  1.00  0.00           H
HETATM   29  H2  UNL     1       5.320   3.524   0.362  1.00  0.00           H
HETATM   30  H3  UNL     1       5.718   4.009  -1.376  1.00  0.00           H
HETATM   31  H4  UNL     1       5.708  -0.465  -1.932  1.00  0.00           H
HETATM   32  H5  UNL     1       4.193  -2.247  -1.508  1.00  0.00           H
HETATM   33  H6  UNL     1       2.615  -2.977   0.912  1.00  0.00           H
HETATM   34  H7  UNL     1       2.476  -3.632  -1.091  1.00  0.00           H
HETATM   35  H8  UNL     1       0.397  -2.680   1.490  1.00  0.00           H
HETATM   36  H9  UNL     1       0.809  -0.893   3.064  1.00  0.00           H
HETATM   37  H10 UNL     1       2.391  -1.571   2.560  1.00  0.00           H
HETATM   38  H11 UNL     1       3.674   2.064   0.760  1.00  0.00           H
HETATM   39  H12 UNL     1      -1.227  -1.315   2.145  1.00  0.00           H
HETATM   40  H13 UNL     1      -0.439   0.268   2.054  1.00  0.00           H
HETATM   41  H14 UNL     1      -2.913   0.256   1.791  1.00  0.00           H
HETATM   42  H15 UNL     1      -1.920   1.157   0.543  1.00  0.00           H
HETATM   43  H16 UNL     1      -4.214  -1.449   1.003  1.00  0.00           H
HETATM   44  H17 UNL     1      -2.385  -1.633  -1.937  1.00  0.00           H
HETATM   45  H18 UNL     1      -1.726   0.019  -1.833  1.00  0.00           H
HETATM   46  H19 UNL     1      -0.833  -2.607  -0.559  1.00  0.00           H
HETATM   47  H20 UNL     1       0.064  -1.391  -1.606  1.00  0.00           H
HETATM   48  H21 UNL     1      -2.533   1.886  -1.217  1.00  0.00           H
HETATM   49  H22 UNL     1      -4.185   3.726  -1.202  1.00  0.00           H
HETATM   50  H23 UNL     1      -7.253   0.911  -0.385  1.00  0.00           H
HETATM   51  H24 UNL     1      -5.609  -0.866  -0.363  1.00  0.00           H
END
```

tried both with NGLview and tried to upload to http://arose.github.io/ngl/

VMD can. :D

**Comment by hainm** [comment]
Interestingly, NGL can render if using multiple model pdb file.

files: https://github.com/hainm/amber_things/tree/master/things/ngl

**Comment by arose** [comment]
It is a parsing error due to the empty chainname, will fix. Thanks for finding this.

**Comment by josemduarte** [comment]
I'd call that a feature. PDB files without chain names are a bit archaic... I'd vote for a "no fix". Or at least for a parser that detects it and gives back a good error message.

**Comment by hainm** [comment]
uhm if 'no fix', `NGL` should say no to `rdkit`.

**Comment by josemduarte** [comment]
Not sure if I follow, are you saying that rdkit produces files like that?

**Comment by hainm** [comment]
uhm, sorry. I misremembered. The 1st post's pdb is from different program that reading rdkit output.

I've tried with `rdkit` and NGL can render `pdb` file, but can not render `mol2` from `rdkit`.

code

```python
from rdkit import Chem
from rdkit.Chem import AllChem
from rdkit.Chem import Draw
from rdkit import rdBase
print(rdBase.rdkitVersion)

m = Chem.AddHs(Chem.MolFromSmiles('COc1ccc2[C@H](O)[C@@H](COc2c1)N3CCC(O)(CC3)c4ccc(F)cc4'))
AllChem.EmbedMultipleConfs(m, useExpTorsionAnglePrefs=True, useBasicKnowledge=True)
Chem.MolToMolFile(m, "test.mol2")
Chem.MolToPDBFile(m, "test_rdkit.pdb")
```

version: 2015.09.1

**Comment by arose** [comment]
fixed 84c3586640d669a62288315145b4021ba36c82cd (not the parsing was failing, but the automatic chain separation and naming due to it being a single residue)

![screenshot 89](https://cloud.githubusercontent.com/assets/272250/15266313/151d0652-1956-11e6-95cb-9d8c7b930f02.png)

**Comment by arose** [comment]
> I'd call that a feature. PDB files without chain names are a bit archaic... I'd vote for a "no fix". Or at least for a parser that detects it and gives back a good error message.

With such an end user facing app not supporting it is not an option. It would be nice to have some form of file format validation and notification, but that is not a priority for me.
